### PR TITLE
BigAnimal: Running Cloud set up in manual mode

### PR DESCRIPTION
**Pull request by moiznalwalla**
Incorporating Zane's comments.

https://edb-starlight.slack.com/archives/C02JYUCEJV7/p1657095608818099?thread_ts=1657095003.335199&cid=C02JYUCEJV7

### CODE REVIEW

**Comment by moiznalwalla** [comment]
@zozowell  @xugy99  Can you review these on priority so that they can go out with the changes related to the deprecation of the script for SPN.

File: product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/01_connecting_your_own_cloud.mdx [hunk 1]
```diff
@@ -77,6 +77,9 @@ Perform the following steps:
    biganimal config set interactive_mode on
    biganimal setup-csp
    ```
+
+!!! Tip
+    Your organization might require you to review the scripts that are invoked while setting up your cloud account. To generate the scripts invoked by `setup-csp`, execute the command without the `--run` option. You can now review and execute `ba-csp-preflight` and then `ba-csp-setup` from your working directory.
```
[the remark this answers is not in the document]
Comment (moiznalwalla) [reply product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/01_connecting_your_own_cloud.mdx]: modifying to 
"Your organization might require you to review the scripts that are invoked while setting up your cloud account. To generate the scripts invoked by `setup-csp`, execute the command without the `--run` option. You can now review and **manually** execute `ba-csp-preflight` and then `ba-csp-setup` from your working directory."

File: product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/index.mdx [hunk 2]
```diff
@@ -81,27 +82,25 @@ Perform the following steps:
...
-    You need to configure your cloud, if any of the readiness checks are not met, see [Configure your Azure subscription](/biganimal/release/getting_started/preparing_cloud_account/01_preparing_azure#configure-your-azure-subscription) or [Configure your AWS account](/biganimal/release/getting_started/preparing_cloud_account/02_preparing_aws/#configure-your-aws-account).
 	
 1. If the cloud readiness checks pass, your cloud account is successfully set up. Connect your cloud account to BigAnimal with following command. 
    ```shell
    biganimal connect-csp --provider  <cloud-service-provider> 
    ```
-
-Once your cloud account is successfully connected to BigAnimal, you and, other users with the correct permissions, can  create clusters.
\ No newline at end of file
+ Once your cloud account is successfully connected to BigAnimal, you and, other users with the correct permissions, can  create clusters.
```
Comment (drothery-edb) [inline product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/index.mdx:106]:
```suggestion
 Once your cloud account is successfully connected to BigAnimal, you, and other users with the correct permissions, can  create clusters.
```
Fixing my own typo

Comment (moiznalwalla) [reply product_docs/docs/biganimal/release/getting_started/02_connecting_to_your_cloud/index.mdx]: Saw and fixed another one.

**Review by drothery-edb — DISMISSED** [review]

**Review by drothery-edb — APPROVED** [review]

**Comment by moiznalwalla** [comment]
@zozowell i have not received any more feedback from you and am assuming you are OK with approving this PR. I will continue to merge these changes. Incase of more changes, I will create a separate PR and ticket.

**Pull request merged**